runtime/doc/lua.txt

```diff
@@ -1541,6 +1541,25 @@ schedule_wrap({cb})                                      *vim.schedule_wrap()*
       • |vim.schedule()|
       • |vim.in_fast_event()|
 
+tr({str}, {opts})                                                   *vim.tr()*
+    Translate keycodes.
+
+    Parameters: ~
+      • {str}   string String to be converted.
+      • {opts}  table|nil Optional arguments:
+                • do_lt: boolean|nil (default: true) Also translate <lt>.
+                  Ignored if `special` is false.
+                • special: boolean|nil (default: true) Replace |keycodes|,
+                  e.g. <CR> becomes a "\r" char.
+                • from_part: boolean|nil (default: true) Legacy Vim parameter.
+                  Usually true.
```

Comment:
I'd recommend removing this from docs, because the effect of this argument is very confusing.